quantum/split_common/transport.c

```diff
@@ -145,6 +156,7 @@ typedef struct _Serial_m2s_buffer_t {
 #    ifdef WPM_ENABLE
     uint8_t current_wpm;
 #    endif
+    uint8_t modifiers;
```

Comment:
```suggestion
    uint8_t real_mods;
    uint8_t weak_mods;
#    ifndef NO_ACTION_ONESHOT
    uint8_t oneshot_mods;
#    endif
```
And this should be moved up above `backlight_level`.

Reply:
I have implented this in my PR